src/screens/Assistant_v2/Results.tsx

```diff
@@ -112,7 +112,7 @@ const Results: React.FC<Props> = ({
   return (
     <View style={styles.container}>
       {tripPatterns?.map((tripPattern, i) => (
-        <Fragment key={String(tripPattern.id ?? i)}>
+        <Fragment key={String(tripPattern.id + i ?? i)}>
```

Comment:
Er det nødvendig å concat `+ i`? burde ikke `tripPattern.id` være unik? Og vil ikke litt poenget med reconciliation av treet gå bort hvis en bruker index på dette viset?
 
```suggestion
        <Fragment key={String(tripPattern.id + i ?? i)}>
```

Reply:
jeg får duplikater her, men jeg har ikke sett på hvorfor.

Reply:
En trippattern.id består av fromQuay, toQuay, when (optional), og servicejourneyId[] (optional).
Jeg antar vi kan få duplikater ved gjentatte spørringer hvis entur returnerer rene gå-strekninger, der from og to er eneste data.
fallbacken til `?? i` eksisterte også i Reisesøk v1.
Er det i det hele tatt sannsynlig at elementer vil forsvinne, endre rekkefølge eller på annet vis kreve en true Id her?